en/docs/learn/user-managed-access.md

```diff
@@ -34,11 +34,11 @@ The Protection API consists of three endpoints:
 </colgroup>
 <tbody>
 <tr class="odd">
-<td><strong><a href="User-Managed-Access-Endpoints_103329668.html#UserManagedAccessEndpoints-Resourceregistrationendpoint">Resource registration endpoint</a></strong></td>
+<td><strong><a href="../../using-wso2-identity-server/user-managed-access-endpoints#resource-registration-endpoint">Resource registration endpoint</a></strong></td>
```

Comment:
this link path should be ../../**learn**/user-managed-access-endpoints#resource-registration-endpoint

Reply:
fixed.